lms/templates/admin/base_site.html

```diff
@@ -20,3 +20,78 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header|default:_('D
 
 
 {% endblock %}
+
+{% block survey_report_banner %}
```

Comment:
Was `base.html` created in order to put a `survey_report_banner` block that could be overridden here? If so, could we append to an existing block using `block.super` instead? Like:

```
{% block header %}{{ block.super }}
  {% include "survey_report/admin_banner.html" %}
{% endblock %}
```

Reply:
That's correct that was the main reason. I'll create a new file containing the banner and test the changes